extensions/shared.webpack.config.js

```diff
@@ -9,11 +9,16 @@
 'use strict';
 
 const path = require('path');
+const fs = require('fs');
 const merge = require('merge-options');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-
+const { NLSBundlePlugin } = require('vscode-nls-dev/lib/webpack-bundler');
 
 module.exports = function withDefaults(/**@type WebpackConfig*/extConfig) {
+	const folderName = path.relative(__dirname, extConfig.context).split(/[\\\/]/)[0];
+	const pkgPath = path.join(__dirname, folderName, 'package.json');
```

Comment:
Maybe simplify: `extConfig.context` is always the `__dirname` of the extension in question (not the dir of the shared config file) and I would have assumed that `join(extConfig.context, 'package.json')` does the job

Reply:
Not in the cases that `extension.webpack.config.js` is deep in some `server` folder, unfortunately...

Reply:
Oh, true. You need the outer most package.json... Bummer. Might deserve a doc comment then

Reply:
Damn I just committed that directly in GitHub using the suggested changes feature. I'm living in the year 3000.